apps/src/code-studio/progress.js

```diff
@@ -12,29 +12,21 @@ import MiniView from './components/progress/MiniView.jsx';
 import DisabledBubblesModal from './DisabledBubblesModal';
 import DisabledBubblesAlert from './DisabledBubblesAlert';
 import {getStore} from './redux';
-import {authorizeLockable} from './stageLockRedux';
```

Comment:
Note: The diff for this file is best reviewed side-by-side (it's very hard to read otherwise)